### PR TITLE
Yasirah Krueng

### PR DESCRIPTION
**Pull request by ykrueng**

### CODE REVIEW

**Comment by juliankohlman** [comment]
Hi Yasirah, I'm Julian a PM for the CSPT2 section, and I'll be reviewing your Sprint.

File: lambdatimes/src/components/Content/Tab.js [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
+  const {tab, selectedTab, selectTabHandler} = {...props};
   return (
-    <div
-      className={''}
-      onClick={() => {
-        /* Replace this dummy click handler function with your selectTabHandler function from props 
-         you'll need to pass the `tab` in as an argument to this handler. */
-      }}
+    <TabComp
+      className={`tab ${tab === selectedTab ? 'active-tab' : ''}`}
```
Comment (juliankohlman) [inline lambdatimes/src/components/Content/Tab.js]: This is a great usage of a ternary here Yasirah 👍 very clean.

File: lambdatimes/src/components/Content/Tabs.js [hunk 2]
```diff
@@ -1,16 +1,70 @@
+// Importing libraries
```
Comment (juliankohlman) [inline lambdatimes/src/components/Content/Tabs.js]: Great job using comments to better organize your imports.

File: lambdatimes/src/components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      }))
+      window.location.reload()
+    } else {
+      this.setState({
+        hint: true,
+      });
+    }
+  }
+
+  handleKey(e) {
```
Comment (juliankohlman) [inline lambdatimes/src/components/Login/Login.js]: Good feature addition here. It's very frustrating as a user when pressing enter does _not_ submit a form or register your input 👍

**Comment by ykrueng** [comment]
> Hi Yasirah, I'm Julian a PM for the CSPT2 section, and I'll be reviewing your Sprint.

Hi Julian, thank you for reviewing my code.

**Pull request closed without merging**